src/Stack/Build/Execute.hs

```diff
@@ -958,7 +958,7 @@ ensureConfig newConfigCache pkgDir ExecuteEnv {..} announce cabal cabalfp task =
             logInfo ""
             logInfo "  stack exec pacman -- --sync --refresh autoconf"
             logInfo ""
-            logInfo $ "Some versions of perl from MYSY2 are broken. See " <>
+            logInfo $ "Some versions of perl from MSYS2 are broken. See " <>
```

Comment:
brand

Reply:
This is fixing a typo, but I agree on 'brand' in output. One of the reasons I introduced a glossary (https://docs.haskellstack.org/en/stable/glossary/) was to try to conform to a consistent usage everywhere in documentation and output.